mw-mobile/lib/routes/routes.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import '../screens/welcome_screen.dart';
+import '../screens/all_ways_screen.dart';
+import '../screens/personal_area_screen.dart';
+import '../screens/all_users_screen.dart';
+import '../screens/home_screen.dart';
+import '../screens/settings_screen.dart';
+import '../screens/about_project_screen.dart';
+
+class AppRoutes {
+  static const String welcome = '/welcome';
+  static const String allWays = '/all_ways';
```

Comment:
Is it conventional for dart to use snake_case for naming?  It will be nice to use the same names as on the web - with camelCase. What do you think?